java/src/main/java/ai/rapids/cudf/RmmAllocationMode.java

```diff
@@ -32,4 +32,8 @@
    * Use arena suballocation strategy
    */
   public static final int ARENA = 0x00000004;
+  /**
+   * Use aligned resource adapter for allocation
+   */
+  public static final int ALIGNED = 0x00000008;
```

Comment:
Do we need this?  Wondering why we couldn't trigger based on allocationAlignment != 0, or is there a use-case where we want the alignment adapter but the requested alignment == 0?

Reply:
Done.